audit/audit-gcp.sh

```diff
@@ -98,6 +98,31 @@ gcloud \
     | sed 's/.googleapis.com//' \
     | while read -r SVC; do
         case "${SVC}" in
+            bigquery)
+                mkdir -p "projects/${PROJECT}/services/${SVC}"
+                rm -f projects/${PROJECT}/services/${SVC}/bigquery.datasets.* || :
+                bq \
+                    --format=json --project_id=$PROJECT ls
+                    > "projects/${PROJECT}/services/${SVC}/bigquery.datasets.json"
```

Comment:
`| jq` just for pretty formatting?  I notice BQ JSON is not indented..

Reply:
well there is --prettyjson